src/Application/src/CalculatorApplication.cpp

```diff
@@ -8,103 +8,65 @@
 {
-    //*************/
-    // Calculator public methods /
-    ///
-    float CalculatorApplication::add (float number1, float number2)
-    {
-        float result = number1 + number2;
-        return result;    
-    }
-    void CalculatorApplication::calculate()
+
+    //function run from main
+    int CalculatorApplication::calculate(float a, float b, char op)
     {
-        float number1, number2;
-        char op;
-        std::cout << CalculatorStrings::WELCOME_MESSAGE << std::endl;
-        std::cout << CalculatorStrings::ENTER_EQUATION << std::endl;
-        std::cin >> number1 >> op >>  number2;
-        while(1)
+        calculator::CalculatorApplicationFactory calculatorAppFactory;  
+        
+        if(limitCheck(a) && limitCheck(b))
         {
-            if(std::cin.fail())
+            auto calculator = calculatorAppFactory.createCalculator(a, b, op); //variables placed into createCalc function and placed into calculator variable
+            if(calculator)
             {
-                std::cin.clear();
-                std::cin.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
-                std::cout << CalculatorStrings::ERROR_MESSAGE_INVALID_INPUT << std::endl;
-                std::cout << CalculatorStrings::RE_ENTER_EQUATION << std::endl;
-                std::cin >> number1 >> op >> number2;
+                std::cout << calculator->toString() << std::endl; //if calculator returned from function is valid, point to string function and output expression and result
             }
-            if(!std::cin.fail())
+            else
             {
-                break;
+                std::cout << CalculatorStrings::ERROR_MESSAGE_INVALID_INPUT << std::endl;
             }
         }
-        CalculatorApplication calculator;
-        switch(op)
+        else
         {
-            case '+':
-            {
-                std::cout << number1 << op << number2 << " = " << calculator.add(number1, number2) << std::endl;
-                break;
-            }
-            case '*':
-            {
-                std::cout << number1 << op << number2 << " = " << calculator.multiply(number1, number2) << std::endl;
-                break;
-            }
-            case '/':
-            {
-                std::cout << number1 << op << number2 << " = " << calculator.divide(number1, number2) << std::endl;
-                break;
-            }
-            case '-':
-            {
-                std::cout << number1 << op << number2 << " = " << calculator.subtract(number1, number2) << std::endl;                
-                break;
-            }
-            case '%':
-            {
-                std::cout << number1 << op << number2 << " = " << calculator.modulus(number1, number2) << std::endl;
-                break;
-            }
-            default:
-            {
-                std::cerr << CalculatorStrings::ERROR_MESSAGE_UNKNOWN_OPERATOR << std::endl;
-            }
+            return -1;
```

Comment:
Consider making an enum, while widely understood -1 and 0 here are magic numbers, e.g.
```cpp
enum EXIT_CODES
{
SUCCESS,
FAIL
}
```